lib/model/vertex.js

```diff
@@ -62,8 +104,24 @@ var Vertex = createClass({
       initOnly: true
     },
 
+    /**
+     * Key/value pairs of styles
+     * Guaranteed to have default styles set
+     * (which come from the static DEFAULT_STYLES on Vertex)
+     * @attribute styles
+     * @type      Object
+     */
     styles: {
-      //dunno
+      validator: Lang.isObject,
+      valueFn: function() {
+        return Vertex.DEFAULT_STYLES;
+      },
+      
+      setter: function(val) {
+        // merge set styles w/ defaults
+        // so that it's guaranteed to have defaults set
+        return ObjectUtils.merge(Vertex.DEFAULT_STYLES, val);
+      }
```

Comment:
That was what I meant by my second option. I can see the advantage of that approach, but also it might cause a problem if we wanted to add a UI for applying styles to elements (other than through a text panel with some GSL in).